plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -145,6 +145,141 @@ func NoDiskConflict(pod *api.Pod, existingPods []*api.Pod, node string) (bool, e
 	return true, nil
 }
 
+type MaxPDVolumeCountChecker struct {
+	filter     PDVolumeFilter
+	maxVolumes uint
+	pvInfo     PersistentVolumeInfo
+	pvcInfo    PersistentVolumeClaimInfo
+}
+
+// Information for how to filter PD Volumes when checking PD Volume caps
+type PDVolumeFilter struct {
+	// Filter normal volumes
+	FilterVolume           func(vol *api.Volume) (id string, relevant bool)
+	FilterPersistentVolume func(pv *api.PersistentVolume) (id string, relevant bool)
+}
+
+// MaxPDVolumeCountPredicate evaluates whether a pod can fit based on the number of volumes which match
```

Comment:
The godoc convention is that you start your doc w/the name of the item in question.